assets/src/components/detoursTable.tsx

```diff
@@ -37,7 +37,7 @@ export const DetoursTable = ({ data }: DetoursTableProps) => {
               {detour.intersection}
             </td>
             <td className="align-middle u-hide-for-mobile">
-              {timeAgoLabel(epochNowInSeconds, detour.activeSince)}
+              {timeAgoLabel(epochNowInSeconds, detour.updated_at)}
```

Comment:
I think that `updated_at` being called here shows that there's a missing translation layer somewhere around `api.ts` and `models/`, since we typically want JS fields to use `camelCase` instead of `snake_case`.

I'm happy to pair on adding in that layer if you'd like (probably tomorrow because my work environment is not conducive to pairing today).

Reply:
👍 ✅ done